src/Components/IncidentList/IncidentCard.Component.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class IncidentCard extends Component {
```

Comment:
Is it not better to use a function since no change of state is being handled/bound?

Reply:
@Mnickii I used class component here since it's later going to be managing state.